docs/section5/Rule5-9.md

```diff
@@ -0,0 +1,45 @@
+
+# Envelope - Rule 5-9  
+
+**Rule ID:** 5-9  
+**Rule Description:**  Vertical Fenestration Assemblies. Fenestration SHGC's for new buildings, existing buildings, and additions shall match the appropriate requirements in Tables G3.4-1 through G3.4-8 using the value for SHGCall for the applicable vertical glazing percentage.  
+**Rule Assertion:** Baseline RMR = expected value  
+**Appendix G Section:** Section G3.1-5(d) Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** Tables G3.4-1 to G3.4-8  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** Tables G3.4-1 to G3.4-8  
+
+## Rule Logic:  
+
+- Get building climate zone: ```climate_zone = B_RMR.weather.climate_zone```  
+
+- For each building segment in the Baseline model: ```for building_segment_baseline in B_RMR.building.building_segments:```  
+
+  - For each thermal_block in building segment: ```for thermal_block_baseline in building_segment_baseline.thermal_blocks:```  
+
+  - For each zone in thermal block: ```for zone_baseline in thermal_block_baseline.zones:```  
+
+  - For each space in thermal zone: ```for space_baseline in zone_baseline.spaces:```  
+
+    - Get surface in space: ```for surface_baseline in space_baseline.surfaces:```  
+
+      - Check if surface is exterior wall: ```if ( surface_baseline.classification == "WALL" ) AND ( surface_baseline.adjacent_to == "AMBIENT" ): wall_surface_baseline = surface_baseline```  
+
+        - Calculate the total wall area: ```total_wall_area_baseline += wall_surface_baseline.area```  
+
```

Comment:
I clarified this approach with the rest of the team - the WWR has to be determined for the entire building, not for each individual surface.
To determine the baseline WWR for the purpose of defining the baseline window U/SHGC, we need to sum up the total exterior wall area - for all above grade walls, and the total window area for all windows on those walls. This will give us the WWR for the entire building, which should be used for determining the U Factor, which should be assigned to all baseline windows.